README.md

```diff
@@ -124,22 +124,22 @@ CloudGraph AWS Provider will ask you what regions you would like to crawl and wi
 | managedAirflow              | iamRole, securityGroups, subnet, s3                                                                                                                                                                                                                                                                                              |
 | nacl                        | vpc                                                                                                                                                                                                                                                                                                                              |
 | natGateway                  | networkInterface, subnet, vpc                                                                                                                                                                                                                                                                                                    |
 | networkInterface            | ec2, eip, efsMountTarget, natGateway, sageMakerNotebookInstance, subnet, vpc, flowLog                                                                                                                                                                                                                                            |
 | organization                |
 | rdsCluster                  | appSync, rdsClusterSnapshot, rdsDbInstance, securityGroup, iamRole, kms                                                                                                                                                                                                                                                          |
 | rdsClusterSnapshot          | kms, rdsCluster, vpc                                                                                                                                                                                                                                                                                                             |
-| rdsDbInstance               | rdsCluster, securityGroup, vpc, subnet                                                                                                                                                                                                                                                                                           |
+| rdsDbInstance               | rdsCluster, securityGroup, vpc, subnet, cloudwatchLog, kms, iamRole                                                                                                                                                                                                                                                              |
```

Comment:
Do the connections need to be reflected in the other lists? (like the connections for iamRole)